data/aliases.js

```diff
@@ -253,15 +254,21 @@ exports.BattleAliases = {
 	"aegiblade": "Aegislash-Blade",
 	"aegis": "Aegislash",
 	"aero": "Aerodactyl",
+	"azu": "Azumarill",
 	"bliss": "Blissey",
+	"bee": "Beedrill",
 	"chomp": "Garchomp",
+	"clef": "Clefable",
+	"coba": "Cobalion",
 	"cofag": "Cofagrigus",
+	"conk": "Conkeldurr",
 	"cress": "Cresselia",
 	"cube": "Kyurem-Black",
 	"cune": "Suicune",
 	"dnite": "Dragonite",
 	"don": "Groudon",
 	"dogars": "Koffing",
+	"drill": "Excadrill",
```

Comment:
I don't really mind doing this, but I still think that drill is used more than exca for excadrill